resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -967,7 +1017,8 @@ protected void doResolveAll(String inetHost, Promise<List<InetAddress>> promise)
     protected void doResolveAll(String inetHost,
                                 DnsRecord[] additionals,
                                 Promise<List<InetAddress>> promise,
-                                DnsCache resolveCache) throws Exception {
+                                DnsCache resolveCache,
+                                DnsCache resolveCacheFallback) throws Exception {
```

Comment:
we will need to preserve the original method signature as well as otherwise this would be a breaking change

Reply:
done